pkg/filter/filter_test.go

```diff
@@ -375,10 +376,14 @@ func (s *FilterSuite) TestJoin(c *C) {
 func (s *FilterSuite) TestResourceIncludeExclude(c *C) {
 	ssTypeRequirement := ResourceTypeRequirement{Group: "apps", Resource: "statefulsets"}
 	pvcTypeRequirement := ResourceTypeRequirement{Version: "v1", Resource: "persistentvolumeclaims"}
-	ss1 := Resource{Name: "ss1", GVR: schema.GroupVersionResource{Group: "apps", Resource: "statefulsets"}}
-	ss2 := Resource{Name: "specificname", GVR: schema.GroupVersionResource{Group: "apps", Resource: "statefulsets"}}
-	pvc1 := Resource{Name: "pvc1", GVR: schema.GroupVersionResource{Version: "v1", Resource: "persistentvolumeclaims"}}
-	pvc2 := Resource{Name: "specificname", GVR: schema.GroupVersionResource{Version: "v1", Resource: "persistentvolumeclaims"}}
+	ss1 := Resource{Name: "ss1", GVR: schema.GroupVersionResource{Group: "apps", Resource: "statefulsets"},
```

Comment:
My unit-test don't work for this.  No action yet as Shippable will check.

Reply:
I'm running
`go test -v ./pkg/filter -check.f=FilterSuite --check.vv`
and tests pass.